bin/flutter

```diff
@@ -81,7 +81,11 @@ fi
     "$FLUTTER_ROOT/bin/internal/update_dart_sdk.sh"
 
     echo Building flutter tool...
-    (cd "$FLUTTER_TOOLS_DIR"; "$PUB" upgrade --verbosity=error --no-packages-dir)
+    pub_env="flutter_install:$PUB_ENVIRONMENT"
```

Comment:
looks like all the other variables are uppercase, we may want to be consistent.
not that we have a bash style guide other than "not use bash". :-)